Sources/AtlasConnectivity/main.swift

```diff
@@ -3,6 +3,9 @@ import MongoSwiftSync
 
 private let configs = ["ATLAS_REPL", "ATLAS_SHRD", "ATLAS_FREE", "ATLAS_TLS11", "ATLAS_TLS12"]
 private let srvConfigs = configs.map { $0 + "_SRV" }
+/// Currently, almost all of the Atlas test instances are running 3.4 which does not support the new "hello" command.
```

Comment:
with the current work being scoped to set all drivers' minWireVersions to 6, I suspect these clusters will end up being upgraded before too long